docs/layouts/partials/toc.html

```diff
@@ -1,7 +1,16 @@
+{{ $tocClasses := "" }}
+{{ if $.Param "rightNav.hideH3" }}
+  {{ $tocClasses = " hide-h3" }}
+{{ end }}
+
+{{ if $.Param "rightNav.hideH4" }}
+  {{ $tocClasses = printf "%s hide-h4" $tocClasses }}
```

Comment:
I think I'd combine this logic into an if/elseif such that hideH4 automatically also hides H3. The flow would need to be this:

if [hideH4], set tocClasses to "hide-h3 hide-h4"
else if [hideH3], set tocClasses to "hide-h3"